client/my-sites/upgrades/cart/cart-item.jsx

```diff
@@ -113,7 +113,7 @@ const CartItem = React.createClass( {
 		} else if ( getIncludedDomain( this.props.cartItem ) ) {
 			info = getIncludedDomain( this.props.cartItem );
 		} else if ( isTheme( this.props.cartItem ) ) {
-			info = this.props.selectedSite.domain;
+			info = this.props.selectedSite && this.props.selectedSite.domain;
```

Comment:
I mean, the way it was written before my change was that it was explicitly set to the selected site and not the cart item, so maybe it should be that way?